starter-code/lab-mongoose-movies/routes/celebrities.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const router  = express.Router();
+const Celebrity    = require('../models/celebrity')
+
+/* GET home page */
+router.get('/celebrities', (req, res, next) => {
+
+  Celebrity.find()
+  .then((theThingsWeGet)=>{
+ 
+    res.render('allCelebs', {listOfCelebs: theThingsWeGet});
+  
+  })
+  .catch((err)=>{
+    console.log(err)
+  })
+  
+
+});
+
+
+router.get('/celebrities/new', (req,res,next)=>{
+  res.render('new');
+})
+
+router.post('/celebrities/create', (req,res,next)=>{
+  // console.log("works here")
+  const celebName = req.body.newName;
+  const celebOcc = req.body.newOccpation;
```

Comment:
this should be `req.body.newOccupation` as in the view you have the input name for this field as `newOccupation`